src/test/groovy/pl/tomaszdziurko/jvm_bloggers/view/login/LoginPageSpec.groovy

```diff
@@ -28,30 +41,68 @@ class LoginPageSpec extends SpringContextAwareSpecification {
         tester = new WicketTester(wicketApplication)
     }
 
-    def "Should redirect to Admin Dashboard after successful login"() {
+    Should "Redirect to Admin Dashboard after successful login"() {
```

Comment:
If it is this project convention than I am ok with it...generally "Should" in a method name is redundant and can be replaced as I did it.